src/sidebar/templates/annotation.html

```diff
@@ -185,6 +185,23 @@
           on-close="vm.showShareDialog = false">
         </annotation-share-dialog>
       </span>
+      <span class="annotation-flag-wrapper " ng-if="vm.isThirdPartyUser()">
```

Comment:
It doesn't look like we actually style this class anywhere. In that case we can remove `annotation-flag-wrapper` I think?